### ISSUE TITLE
Reload button n Icons !

### ISSUE THREAD

**Issue by deepaksp** [Shihira/gnome-extension-quicktoggler#4]
it will cool if extension can have this 2 options

**Comment by hitex** [comment]
You can easily add "Reload" button yourself:

``` json
{
    "type": "launcher",
    "title": "Restart Extension",
    "command": "gnome-shell-extension-tool -d quicktoggler@shihira.github.com && gnome-shell-extension-tool -e quicktoggler@shihira.github.com"
}
```

**Comment by deepaksp** [comment]
yes n what about icons ??

**Comment by Shihira** [comment]
I feel little confused about what the 'reload' exactly means. Does that mean you modified configuration file and the indicator icon settings, and hope for a instant functionality to apply these modifications?

**Comment by deepaksp** [comment]
yup .. suppose i have created a new nginx host config file, mostly we need to reload nginx to take effect .. it will be cool if it can track config files.

though thr are so  many ways ! i know n i am using it but thought it will be cool.

**Comment by Shihira** [comment]
@deepakSP Ok...Spying preference and configuration. I have to recognize that it's quite useful, while it requires quite heavy work. I will try to implement that. Please keep watching.

Lastly I want to thank you for your using and supporting this extension. Your support is the motivation of me to continue the development.

**Comment by deepaksp** [comment]
don't thank me dude .. you are doing great job keep it up . n ya thanks for awesome extension :D

**Comment by Shihira** [comment]
@deepakSP In lastest commits(e69c7efb63d58a60d8123b1f53c0e3ed0180ec5f) configuration and preference auto reloading has been implemented. Have a try and report to me if there are any bugs!
